### ISSUE TITLE
Questions want to be guided

### ISSUE THREAD

**Issue by yangmin32** [Sriram94/pomfrl#1]
Hello, I was fortunate to read your article, but I encountered a problem in code reproduction. I have tried many methods and failed to compile build.sh successfully. However, the original Magent can be compiled successfully. I would like to ask about the following situation what is the problem? Hope to get your guidance, thank you very much!
```
[90%] Linking cxX shared library libnagent.so
[90%] Built target magent
[93%] Building cxx object CMakeFiles/testlib.dir/src/utility/utility.cc.o
[96%] Linking cxx executable testlib
[96%] Built target testlib
CMakeFiles/Hakefile2:67:recipe for target'CNakeFiles/render.dir/all' failed
make[1]:***[CMakeFiles/render.dir/all]Error 2
Makefile:83: recipe for target'all' failed
make:***[all] Error 2
```

**Comment by Sriram94** [comment]
Sorry, I am not able to reproduce this error. The build.sh is a part of the MAgent library. Recently I observed that the type of hardware used can have an impact on the MAgent library and the library has problems in certain systems (as seen in the issues [here](https://github.com/geek-ai/MAgent/issues/89), [here](https://github.com/geek-ai/MAgent/issues/51) and [here](https://github.com/Farama-Foundation/PettingZoo/issues/284)), but it is not very clear what exactly is causing the issue and in which system(s) the problem appears. 

Since the library is already compiled can you continue with the other steps and try to run the training program? If you encounter an error somewhere, please send that error message here, and I will see if I can reproduce that error or find a fix for that.

**Comment by yangmin32** [comment]
Thank you for your reply, I have tried to run the training program, and the following error appears, maybe it is the problem of build.sh? It seems that there is no good solution yet.

```
[*] ROUND #0, EPS: 1.00 NUMBER: [25, 25]
terminate called after throwing an instance of 'std::runtime_error'
  what():  /tmp/pip-req-build-r9dc3mo7/src/gridworld/GridWorld.cc:892 : unsupported info name in GridWorld::get_info : hp
已放弃 (核心已转储)
```

**Comment by Sriram94** [comment]
Thank you for providing the error and bringing this to my notice. I will try to find a solution and respond as soon as I can. I am expecting that it will take some time since I am still not able to reproduce this error. Apologies for the delay in advance.

**Comment by yangmin32** [comment]
Very grateful for your help！

**Comment by Sriram94** [comment]
Can you email s2ganapa@uwaterloo.ca a screenshot of your complete console output as you run all the installation commands as given in the README? Also do mention your OS and CPU architecture.

**Comment by Sriram94** [comment]
Fixed by commit 979ce615828cc4bed086152917885752d2a1bece.
